components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -6817,6 +6817,45 @@ paths:
       operationId: getSettings
 
   ######################################################
+  # The linter custom rules resource APIs
+  ######################################################
+  /linter-custom-rules:
+    get:
+      tags:
+        - Linter Custom Rules
+      summary: Get linter custom rules.
+      description: |
+        This operation can be used to get linter custom rules from tenant-config.
+      operationId: getLinterCustomRules
+      responses:
+        200:
+          description: |
+            OK.
+            Linter Custom Rules Retrieved Successfully.
+          headers:
+            Content-Type:
+              description: |
+                The content type of the body.
+              schema:
+                type: string
+          content:
+            application/json:
+              schema:
+                type: string
```

Comment:
Can't we introduce a schema for this instead of a string? Can you provide an example json payload here so we can decide the feasibility.

Reply:
The response payload is a set of rules defined according to spectral custom rules definition ([sh api-manager.sh -Dmigrate -Dcomponent=identity](https://meta.stoplight.io/docs/spectral/e5b9616d6d50c-custom-rulesets)). Therefore, the schema will be dynamic.